sdk/tests/conformance2/textures/misc/copy-texture-image.html

```diff
@@ -0,0 +1,225 @@
+
+function enumToString(value) {
+    return wtu.glEnumToString(gl, value);
+}
+
+function checkFramebuffer(expected) {
+    var actual = gl.checkFramebufferStatus(gl.FRAMEBUFFER);
+    if (expected.indexOf(actual) < 0) {
+        var msg = "checkFramebufferStatus expects [";
+        for (var index = 0; index < expected.length; ++index) {
+            msg += wtu.glEnumToString(gl, expected[index]);
+            if (index + 1 < expected.length)
+                msg += ", ";
+        }
+        msg += "], was " + wtu.glEnumToString(gl, actual);
+        testFailed(msg);
+    } else {
+        var msg = "checkFramebufferStatus got " + wtu.glEnumToString(gl, actual) +
+                  " as expected";
+        testPassed(msg);
+    }
+}
+
+var testInternalformat = function () {
+    var goodByteFormats = [
+        "RGB",
+        "RGBA",
+        "LUMINANCE_ALPHA",
+        "LUMINANCE",
+        "ALPHA",
+        "R8",
+        "RG8",
+        "RGB565",
+        "RGB8",
+        "RGBA4",
+        "RGB5_A1",
+        "RGBA8",
+        "RGB10_A2",
+    ];
+    var goodSRGBFormats = [
+        "SRGB8",
+        "SRGB8_ALPHA8",
+    ];
+    var goodIntFormats = [
+        "R8I",
+        "R16I",
+        "R32I",
+        "RG8I",
+        "RG16I",
+        "RG32I",
+        "RGBA8I",
+        "RGBA16I",
+        "RGBA32I",
+    ];
+    var goodUnsingedIntFormats = [
+        "R8UI",
+        "R16UI",
+        "R32UI",
+        "RG8UI",
+        "RG16UI",
+        "RG32UI",
+        "RGB10_A2UI",
+        "RGBA8UI",
+        "RGBA16UI",
+        "RGBA32UI",
+    ];
+    var badByteFormats = [
+        "R8_SNORM",
+        "RG8_SNORM",
+        "RGB8_SNORM",
+        "RGBA8_SNORM",
+    ];
+    var badIntFormats = [
+        "RGB8I",
+        "RGB16I",
+        "RGB32I",
+    ];
+    var badUnsingedIntFormats = [
+        "RGB8UI",
+        "RGB16UI",
+        "RGB32UI",
+    ];
+    var badFloatFormats = [
+        "R16F",
+        "R32F",
+        "RG16F",
+        "RG32F",
+        "R11F_G11F_B10F",
+        "RGB9_E5",
+        "RGB16F",
+        "RGB32F",
+        "RGBA16F",
+        "RGBA32F",
+    ];
+    var badDepthStencilFormats = [
+        "DEPTH_COMPONENT16",
+        "DEPTH_COMPONENT24",
+        "DEPTH_COMPONENT32F",
+        "DEPTH24_STENCIL8",
+    ];
+
+    var texture = gl.createTexture();
+    gl.bindTexture(gl.TEXTURE_2D, texture);
+
+    function testFormat(internalformat, srcTexFormatsTypes, fboAttachmentType, expected, msg) {
+        var fbo = gl.createFramebuffer();
+        var srcTexture = gl.createTexture();
+        gl.bindTexture(gl.TEXTURE_2D, srcTexture);
+        gl.texImage2D(gl.TEXTURE_2D, 0, srcTexFormatsTypes.internalformat, 64, 64, 0, srcTexFormatsTypes.format, srcTexFormatsTypes.type, null);
+        gl.bindFramebuffer(gl.FRAMEBUFFER, fbo);
+        gl.framebufferTexture2D(gl.FRAMEBUFFER, fboAttachmentType, gl.TEXTURE_2D, srcTexture, 0);
+        checkFramebuffer([gl.FRAMEBUFFER_COMPLETE]);
```

Comment:
Actually it is still spec compliant if the underlying driver does not support a combination of color/depth/stencil.

So what should be done here if to check if the framebuffer is complete.  If not, return early, but it's not a test failure.